### DISCUSSION TITLE
Feature request: Filter for untracked issues

### DISCUSSION THREAD

**Discussion by thclark** [community/community#37023] [Projects and Issues]
### Current situation

When an issue is mentioned by another issue, it shows up as being 'Tracked by'. The tracked issue looks like this:

<img width="790" alt="Screenshot 2022-10-23 at 11 51 42" src="https://user-images.githubusercontent.com/7223170/197388014-2036dd36-c40c-4955-96cd-2a01ef6d5b7f.png">

However, in the [filtering and searching documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests) there doesn't seem to be a way of filtering for issues which are trackers/trackees/untracked.

Is it possible to filter for issues which are not tracked by another issue?

### Use Case

The new "tracking" functionality with GitHub allows me to create epics and the powerful new project boards mean we can manage basically everything without a third party issue tracking solution.

However, I want to make sure that all my issues either *are* epics, or *belong to* epics:
- so they don't get lost,
- so my product manager and team members can see how an individual issue fits into our wider strategic context, and
- so I know when to prioritise them as we shift our development focus over time. 

Filtering for issues that aren't labeled `epic` is easy (achievable using `label:-epic`). I need to also filter for issues which aren't `tracked` (because if they're not tracked they need to be added to an epic).

### Suggested solution

Being able to filter for: `is:tracked` (and its negated form `-is:tracked`) would solve this. Complimentary `is:tracker` and `-is:tracker` might be a sensible pairing to make this area of filtering feature complete.

**Comment by ossanna16** [comment]
Hi there @thclark and welcome to our community 😄. Thanks so much for bringing this up!
